unittests/PhasarLLVM/DataFlowSolver/Mono/InterMonoTaintAnalysisTest.cpp

```diff
@@ -77,6 +78,8 @@ class InterMonoTaintAnalysisTest : public ::testing::Test {
       FoundResults.insert(getMetaDataID(Result));
     }
     EXPECT_EQ(FoundResults, GroundTruth);
+
+    delete PT;
```

Comment:
Fixing the leak here is a good idea :smile: , however, I think we should just replace the call to `new` with a `make_unique` and use `unique_ptr` instead.